app/controllers/site_controller.rb

```diff
@@ -417,6 +417,9 @@ def view_gene_expression
     @cluster_annotations = load_cluster_group_annotations
     @top_plot_partial = @selected_annotation[:type] == 'group' ? 'expression_plots_view' : 'expression_annotation_plots_view'
     @y_axis_title = load_expression_axis_title
+    matrix = @study.expression_matrix_files.first
```

Comment:
Non-blocking refinement: while this will give you a matrix from which you can extract the `taxon`, a nice enhancement might be to extract that from each matrix (rather than just the first) and store in an object that you could use in the front end in a select (in case the study has multiple species).  For example:

```
@taxons = []
@study.expression_matrix_files.each do |matrix|
    @taxons << {taxid: matrix.taxon.ncbi_taxid, organism: matrix.taxon.scientific_name}
end
```

Reply:
Good catch here.  I've stashed [some scratch code to enable this](https://github.com/broadinstitute/single_cell_portal_core/pull/552/commits/990b6abd944b62a1bce121558c3592328ec74de6), but it needs more work.

Tandem changes will be needed for gene autocomplete.  These would preserve current gene autocomplete behavior in Study Overview, while ensuring the correct organism's genome is shown for the selected gene.

Reply:
My impression is that there are relatively painless and risk-free calls (just a couple of DB lookups), but I think it's worth wrapping the additions to this method in a feature flag block just so that we know what they belong to